tests/unit_tests/test_runner.py

```diff
@@ -174,3 +175,59 @@ def test_save_final_model(tmp_path, mgf_small, tiny_config):
 
     assert model_file.exists()
     assert validation_file.exists()
+
+
+def test_evaluate(
+    tmp_path, mgf_small, mzml_small, mgf_small_unannotated, tiny_config
+):
+    """Test model evaluation during sequencing"""
+    # Train tiny model
+    config = Config(tiny_config)
+    config.max_epochs = 1
+    model_file = tmp_path / "epoch=0-step=1.ckpt"
+    with ModelRunner(config) as runner:
+        runner.train([mgf_small], [mgf_small])
```

Comment:
**question:** I don't think training is really necessary for this test, we can just use a randomly initialized model, if that's compatible with our `ModelRunner` interface.

Reply:
Training for one epoch was a workaround in order to get randomly initialized model weights. AFIK it's impossible to initialize the `ModelRunner` with random weights except for during training; in any other case model weights have to be provided.